browser/static/ably.js

```diff
@@ -7693,6 +7703,42 @@ var RealtimeChannel = (function() {
 		this.pendingEvents = [];
 	};
 
+	RealtimeChannel.prototype.history = function(params, callback) {
```

Comment:
This is new - or rather, copied with changes from channel.js to realtimechannel.js. RealtimeChannel had no separate `history` method before. If you want to see the diff between `Channel#history` and `RealtimeChannel#history`, I've put it in a gist: https://gist.github.com/SimonWoolf/e4adee04837da359dc89